extension/scripts/features/faction-quick-items/ttFactionQuickItems.js

```diff
@@ -468,8 +468,19 @@
 		event.stopPropagation();
 		event.preventDefault();
 
-		const target = event.target.dataset.type === "tt-points" ? event.target : findParent(event.target, { tag: "LI" });
-		const id = target.find(".img-wrap").dataset.itemid;
+		let id = null;
+
+		// These are points
+		if (event.target.dataset.role === "give" || event.target.dataset.role === "refill") {
+			if (event.target.innerHTML.includes("Energy")) {
+				id = "points-energy";
+			} else {
+				id = "points-nerve";
+			}
+		} else {
+			const target = findParent(event.target, { tag: "LI" });
+			id = target.find(".img-wrap").dataset.itemid;
+		}

```

Comment:
Shouldn't be needed if you update `setupQuickDragListeners` to work with your page.

Reply:
Got it. Reverted.